src/uwtools/resources/rocoto.jsonschema

```diff
@@ -263,103 +322,41 @@
           "$ref": "#/$defs/timestr"
         }
       },
-      "additionalProperties": false,
       "required": [
         "command",
         "walltime"
       ],
-      "dependentSchemas": {
-        "join": {
-          "not": {
-            "required": [
-              "stdout"
-            ]
-          }
-        },
-        "stdout": {
-          "not": {
-            "required": [
-              "join"
-            ]
-          },
-          "required": [
-            "stderr"
-          ]
-        },
-        "stderr": {
-          "required": [
-            "stdout"
-          ]
-        },
-        "shared": {
-          "not": {
-            "required": [
-              "exclusive"
-            ]
-          }
-        },
-        "exclusive": {
-          "not": {
-            "required": [
-              "shared"
-            ]
-          }
-        }
-      },
-      "oneOf": [
-        {
-          "required": [
-            "cores"
-          ]
-        },
-        {
-          "required": [
-            "native"
-          ]
-        },
-        {
-          "required": [
-            "nodes"
-          ]
-        }
-      ]
-    },
-    "TimePattern": {
-      "type": "string",
-      "pattern": "^(\\d{2}:){3}\\d{2}$"
+      "type": "object"
     },
     "timestr": {
-      "type": "string",
-      "pattern": "^[0-9]{2}:[0-9]{2}:[0-9]{2}$"
+      "pattern": "^[0-9]{2}:[0-9]{2}:[0-9]{2}$",
+      "type": "string"
     },
     "var": {
-      "type": "object",
+      "additionalProperties": false,
+      "minProperties": 1,
       "patternProperties": {
         "^.+$": {
           "type": "string"
         }
       },
-      "additionalProperties": false,
-      "minProperties": 1
+      "type": "object"
     }
   },
-  "type": "object",
   "properties": {
     "workflow": {
-      "type": "object",
+      "additionalProperties": false,
       "properties": {
         "attrs": {
-          "type": "object",
           "properties": {
             "cyclethrottle": {
-              "type": "integer",
-              "minimum": 0
+              "minimum": 0,
```

Comment:
I'm kind of surprised there's so many flips like this even though I've put it through `jq` several times, is it just an additional effect of adding it to `make format` instead of separate?